### PR TITLE
feat(gsd-media-card): add --media-card-height variable

### PR DESCRIPTION
**Pull request by dominickudiabor**
Screenshot 2021-02-01 at 23 44 37

### CODE REVIEW

File: src/components/gds-media-card/gds-media-card.stories.js [hunk 1]
```diff
@@ -90,4 +90,26 @@ export const Examples = () => html`
       <gds-paragraph size="s">which appears after after the other slots</gds-paragraph>
     </div>
   </div>
+  <div style="margin-bottom: 40px; height: 700px; border: 1px dashed black;">
+    <style>
+      .full-height {
+        --card-height: 100%;
+        --media-card-height: 100%;
+        height: 100%;
```
Comment (taromorimoto) [inline src/components/gds-media-card/gds-media-card.stories.js:98]: @dominickudiabor I think you should remove this height: 100%; and add that --gds-link-height: 100%;, right? Or is this working correctly on your local machine?

Comment (dominickudiabor) [reply src/components/gds-media-card/gds-media-card.stories.js:98]: I removed it and its still working correctly.

Comment (taromorimoto) [reply src/components/gds-media-card/gds-media-card.stories.js:98]: What do you mean? :) Screenshot?

File: src/components/gds-link/gds-link.scss [hunk 2]
```diff
@@ -3,9 +3,11 @@
 a {
   @include base;
   display: inline-block;
+  height: var(--gds-link-height);
```
Comment (henritukiainen) [inline src/components/gds-link/gds-link.scss:6]: Should these also have default value `auto`?

File: src/components/gds-card/gds-card.scss [hunk 3]
```diff
@@ -3,6 +3,7 @@
 :host {
   @include card;
   transition: box-shadow 0.3s;
+  height: var(--card-height);
```
Comment (henritukiainen) [inline src/components/gds-card/gds-card.scss:6]: Should this also have default value `auto`?

Comment (taromorimoto) [reply src/components/gds-card/gds-card.scss:6]: @henritukiainen  Yeah, I've been thinking about what is the correct way of doing this "fallback to initial value". Is it like you suggest that we define the initial value, or that we don't define the variable and the value then default to initial value? Does it do that?

Comment (taromorimoto) [reply src/components/gds-card/gds-card.scss:6]: @henritukiainen  I think in the end, setting the fallback value explicitly is probably the better option as it's clear to the developer what is going to be in the value. @dominickudiabor if you add these `auto` fallbacks and then merge?

Comment (dominickudiabor) [reply src/components/gds-card/gds-card.scss:6]: okay!

**Pull request merged**